cli/cmd/metanode.go

```diff
@@ -172,10 +171,6 @@ func newMetaNodeMigrateCmd(client *master.MasterClient) *cobra.Command {
 			}()
 			src = args[0]
 			dst = args[1]
-			if optCount > mpMigrateMax || optCount <= 0 {
-				stdout("Migrate mp count should between [1-15]\n")
```

Comment:
Both here and on the server side should retain it, and use the variable defaultMigrateMpCnt. This variable can be moved to the util module.

Reply:
fix it